uc-crux-llvm/src/UCCrux/LLVM/Classify/Types.hs

```diff
@@ -228,33 +236,38 @@ data Uncertainty
     UMissingAnnotation Crucible.SimError
   | -- | A user assertion failed, but symbolically
     UFailedAssert !What4.ProgramLoc
+  | -- | Simulation timed out
+    UTimeout !Text
   deriving (Show)
 
 partitionUncertainty ::
-  [Uncertainty] -> ([Crucible.SimError], [What4.ProgramLoc], [Panic Unimplemented], [Unclassified], [Unfixed], [Unfixable])
-partitionUncertainty = go [] [] [] [] [] []
+  [Uncertainty] -> ([Crucible.SimError], [What4.ProgramLoc], [Panic Unimplemented], [Unclassified], [Unfixed], [Unfixable], [Text])
+partitionUncertainty = go [] [] [] [] [] [] []
```

Comment:
This looks like a place where a local record data structure would simplify and prevent mixing up the arguments.

Reply:
FWIW, they all have different types. But it would probably help with readability too.

Reply:
https://github.com/GaloisInc/crucible/issues/691